src/index.jsx

```diff
@@ -8,8 +8,8 @@ const defaultSerializers = SanityBlockContent.defaultSerializers
  * Renders an array of Portable Text blocks as React components.
  *
  * @param {object} props
- * @param {[object]} props.content Array of portable text blocks
- * @param {string} props.className Optional className
+ * @param {object[]} props.content Array of portable text blocks
+ * @param {string | undefined} props.className Optional className
```

Comment:
Does it work to instead use the JSDoc syntax for this? VSCode doesn't seem to treat `{string | undefined}` as optional (in a JavaScript context at least).

```suggestion
 * @param {string} [props.className] Optional className
```